client/src/boundary/interactor/GetGameInteractor.ts

```diff
@@ -12,7 +13,9 @@ export class GetGameInteractor implements GetGameUC {
     this.gameConverter = gameConverter;
   }
 
-  getGame(id: string): BoundaryGame {
-    return this.gameConverter.convert(this.gameStorage.getGame(id));
+  getGame(id: string): Observable<BoundaryGame> {
+    return this.gameStorage
+      .getGame(id)
+      .pipe(map((game) => this.gameConverter.convert(game)));
```

Comment:
jeigu vienas parametras funkcijoj tai galima praleist () aplink jį: map(game => this.gameConverter.convert(game))

Reply:
šitoj vietoj tai automatiškai on-save uždeda ()

Reply:
jam automatiškai juos uždeda IDE, tai gali dabar būti taip. Gal kažkada pasakysiu kokį config naudoti, kad būtų kaip pas mus